package.json

```
@@ -1,10 +1,11 @@
 {
 	"name": "caprine",
+	"appName": "caprine",
 	"productName": "Caprine",
 	"version": "2.57.3",
 	"description": "Elegant Facebook Messenger desktop app",
 	"license": "MIT",
-	"repository": "sindresorhus/caprine",
+	"repository": "https://github.com/sindresorhus/caprine",
```

That's not needed. `package.json` supports the [shortcut syntax](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository).
```suggestion
	"repository": "sindresorhus/caprine",
```

---

I know, but it can be good for if a user wants to copy/paste the url or for some node extenders if say, someone wanted to grep the repo url and use it for some reason. npm supports both so I have always made it a habit to just go ahead and use the full url

---

I still don't really think this needs to change. If a user wants to copy/paste a url of a repo while looking at a package.json file, I think they can manage with just the user/repo slug while we get a clean package.json 😁